_docs/oak-js.md

```diff
@@ -29,3 +29,4 @@ async function main() {
 main().catch(console.error).finally(() => process.exit()); 
 ```
 
+You can find more examples in our [Javascript Example Repo](https://github.com/OAK-Foundation/javascript-examples).
```

Comment:
i assume the code above is still applicable? Also can you rm line 17 pls the `//`